charts/corda/values.yaml

```
@@ -288,4 +288,35 @@ workers:
       # -- the CPU/memory resource requests for the p2p-link-manager worker containers
       requests: {}
       # -- the CPU/memory resource limits for the p2p-link-manager worker containers
-      limits: {}
\ No newline at end of file
+      limits: {}
+
+  # P2P Gateway worker configuration
+  p2pGateway:
+    # p2p-gateway worker image configuration
+    image:
+      # -- p2p-gateway worker image registry, defaults to image.registry
+      registry: ""
+      # -- p2p-gateway worker image repository
+      repository: "corda-os-p2p-gateway"
+      # -- p2p-gateway worker image tag, defaults to image.tag
+      tag: ""
+    # -- p2p-gateway worker replica count
+    replicaCount: 1
+    # -- Use stub membership and crypto components

```

nit: I think the gateway only has stubs for the crypto processor
```suggestion
    # -- Use stub crypto processor

```